docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/risk_effects/index.rst

```diff
@@ -35,3 +35,190 @@ Iron Deficiency Risk Effects
+
+Instead, the relative risks should be applied to both the YLD (or incidence) and YLL (CSMR or EMR) rate for maternal disorders (or a specific modeled individual maternal disorder), such that:
+
+.. math:: 
+
+    rate_i = rate_\text{GBD} * 1 / e^{\text{ln(RR)} * (Hgb_i - Hgb_\text{GBD})}
+
+Where, 
+
+.. list-table:: Parameter Definitions
+   :widths: 5 5 5
+   :header-rows: 1
+
+   * - Parameter
+     - Definition
+     - Note
+   * - :math:`Hgb_i`
+     - Hemoglobin value for an individual pregnant simulant in g/dL
+     - Needs unit conversion! (Exposure typically defined in g/L)
+   * - :math:`Hgb_\text{GBD}`
+     - Mean hemoglobin value among the pregnant/lactating population from GBD in g/dL
+     - Needs unit conversion! (Exposure typically defined in g/L)
+   * - :math:`rate_i`
+     - Rate for an individual simulant
+     - Rate can be incidence, EMR, CSMR, etc.
+   * - :math:`rate_\text{GBD}`
+     - Rate from GBD 
+     - Rate can be incidence, EMR, CSMR; consider whether this should be among total population or pregnant population in maternal disorders cause document
+   * - :math:`RR`
+     - Relative risks for iron deficiency and maternal disorders
+     - Should be constant for all age-groups and causes within maternal disorders group, can choose any
+
+.. list-table:: Relative Risks
+   :widths: 5 5 5
+   :header-rows: 1
+
+   * - Exposure
+     - Relative Risk
+     - Note
+   * - Per unit deficit of hemoglobin (g/dL)
+     - 1.252472 (0.920269, 0.701621)
+     - Note unit change (exposure is typically in g/L)
+
+Validation and Verification Criteria
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Parameters that should be tracked for verification and validation include:
+
+- Simulant pregnancy status (stratification preferred)
+- Simulant hemoglobin concentration
+- Simulant maternal disorder state
+
+Simulants with higher hemoglobin concentrations should have lower maternal disorder rates (the specific difference in maternal disorder rates can be quanified for simulants with known hemoglobin concentrations).
+
+If intervention causes increase in hemoglobin, the intervention scenario rate of maternal disorders should be lower than the baseline rate of maternal disorders.
+
+Assumptions and Limitations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Notably, anemia is a sequela of the maternal disorder, maternal hemorrhage. Therefore, if a simulation model of an intervention that affects hemoglobin concentration evaluates both the impact on burden due to maternal disorders and the impact of YLDs due to anemia, it is possible that the impact on YLDs for anemia due to maternal hemorrhage will be double counted. This is likely a relative small portion of DALYs, but should be investigated and considered prior to implementation.
```

Comment:
How about saying "anemia is a sequela of maternal hemorrhage, which is a subcause of maternal disorders" for the first sentence?